proto/klyraprotocol/yields/tx.proto

```
@@ -0,0 +1,12 @@
+
+syntax = "proto3";
+package klyraprotocol.yields;
+
+option go_package = "github.com/StreamFinance-Protocol/stream-chain/protocol/x/yield/types";
+
+import "cosmos/msg/v1/msg.proto";
```

Buf (1.47.2)

import "cosmos/msg/v1/msg.proto": file does not exist

(COMPILE)